src/types/SisoTfTypes/SisoZpk.jl

```diff
@@ -109,32 +103,38 @@ function minreal(sys::SisoZpk{T,TR}, eps::Real) where {T, TR}
     SisoZpk{T, TR}(newZ, newP, sys.k)
 end
 
-# FIXME: Perhaps move to some other file with auxilliary functions,
-# the name could also be imporoved. Perhaps this functionality can be found in some other package.
-""" If TR is Complex and T is Real, check that every pole is matched to its conjugate
-    this assumes that the compelx poles are ordered as they are output by the LAPACK
-    routines that return complex-conjugated values, i.e., (x+iy) is followed by (x-iy)"""
-function check_real(r_vec::AbstractVector{<:Complex})
-    k = 1
-    while k <= length(r_vec)
-        if isreal(r_vec[k])
-            # Do nothing
-        else
-            if k == length(r_vec) # Element is complex and there is no more elements to match up with
-                return false
-            elseif conj(r_vec[k]) == r_vec[k+1] # Element is complex and succeeding element is its conjugate
-                k=k+1
-            else
-                return false
+""" If TR is Complex and T is Real, pair up every pole with its conjugate as they are ordered 
+    by the LAPACK routines that return complex-conjugated values, i.e., (x+iy) is followed by (x-iy)"""
```

Comment:
```suggestion
"""Reorder the vector x of complex numbers so that complex conjugates come after each other, with the one with positive imaginary part first. Returns true if the conjugates can be paired and otherwise false."""
```
Good with a comment, but what is TR and T? Also, might not necessarily be poles.

Reply:
Yeah, mostly just stole the comment from `checkreal` to have something, but yours is much clearer.